examples/koa/client.js

```diff
@@ -13,10 +13,10 @@ function makeRequest() {
   api.context.with(api.trace.setSpan(api.ROOT_CONTEXT, span), async () => {
     try {
       const res = await axios.get('http://localhost:8081/run_test');
-      span.setStatus({ code: api.StatusCode.OK });
+      span.setStatus({ code: api.SpanStatusCode.OK });
```

Comment:
mmm, I think that we should actually delete this line.
span status code should be `UNSET` which is the default.

[from the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status):
> Generally, Instrumentation Libraries SHOULD NOT set the status code to Ok, unless explicitly configured to do so. Instrumentation Libraries SHOULD leave the status code as Unset unless there is an error, as described above.

Reply:
Isn't this example an end-user example?

Reply:
@blumamir would you mind shedding your light on this again? I believe this is an end-user example so setting the status code here should be fine.

Reply:
@legendecas is correct the end user should set the span status and this example is fine IMO